packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -366,6 +390,18 @@ class AndroidCameraCameraX extends CameraPlatform {
     previewInitiallyBound = true;
     _previewIsPaused = false;
 
+    // Retrieve info required for correcting the rotation of the camera preview
+    // if necessary.
+    isUsingSurfaceTextureForPreview =
+        await SystemServices.isUsingSurfaceTextureForPreview();
+    final Camera2CameraInfo camera2CameraInfo =
+        await proxy.getCamera2CameraInfo(cameraInfo!);
+    sensorOrientation = await proxy.getSensorOrientation(camera2CameraInfo);
```

Comment:
nit: It would be nice to do these in parallel since they don't depend on each-other.

Same with `isUsingSurfaceTextureForPreview` above.

As implemented, this `+ ...` adds 4 serial async/await calls to initialization.